### ISSUE TITLE
Add Stacks page under Che's Dashboard

### ISSUE THREAD

**Issue by slemeur** [eclipse-che/che#2206]
Add Stacks list under Che's Administration page

We will add a new tab under the Che's Administration page on the User Dashboard: "Stacks"
In this page we will display the list of all the Stacks that are registered on Che. (ie stacks.json)

![6 - stacks - actions - option2](https://cloud.githubusercontent.com/assets/1636769/17960108/5e7ff3aa-6aa4-11e6-89d8-53487cc0e89b.png)

**Comment by TylerJewell** [comment]
Maybe this should be a primary left nav item called Stacks?  If we are going to do a lot more with stacks sharing and notification among shares users then that would not be an administration issue - it would just be stacks.

**Comment by slemeur** [comment]
OK +1 

Updated design:
![6 - stacks - actions - option3](https://cloud.githubusercontent.com/assets/1636769/17985530/8a386e7e-6b16-11e6-97a4-0799de450741.png)

**Comment by TylerJewell** [comment]
+1 on the location - definitely want this as a primary object.  @slemeur - lets also think about some of the onboarding workflows for new users into the Che product once stack editing is available.

If you are a new user in Che, today we push you to create your first workspace.  We want to get them with a private repository that they can make a commit with.  Should we be thinking a bit more about pushing people into creating their first custom stack?

1: Browsing existing stacks for their contents.  I noticed that many of the stacks have descriptions and component lists that could be updated (version numbers and more complete items).

2: Browsing the different stacks by operating system.  I noticed that we do not have a filter component on the ready-to-go stacks, but we do on the stack library.  We should be consistent.

3: How will changes to stacks have notifications to users?  This is a super tricky issue as we can track changes to the meta information, but we will not necessarily be able to track changes to internal images.

4: When users create new stacks, creating from an existing one or a new from a template will be a big deal.  We will want to make it really fun for people to create new stacks over time.

Ok to start breaking these issues into different issues - just starting the discussion.
